packages/evolution-common/src/services/baseObjects/attributeTypes/PersonAttributes.ts

```diff
@@ -70,25 +76,98 @@ export type RidesharingUser = YesNoDontKnowNonApplicable;
 
 // A person can have multiple occupations, like fullTimeStudent + partTimeWorker:
 export const occupationValues = [
```

Comment:
For occupationValues, 

I have a question. For one survey (MJ), we asked this question with multiple choices (InputCheckbox). The respondent can choose student and worker status. So do we need all these choices?

Reply:
These are all the possible values, it does not mean we ask them in an input. When we define each class, we must include every value that could be in each attributes. When we will create the input themselves, we can then choose/filter values accordingly.

Reply:
Why not allow an array then?

Reply:
Never seen occupation as an array in any survey. Using an array would mean long discussions... You can add an issue for the future though.